### PR TITLE
Create a script to update AUTHORS.

### PR DESCRIPTION
**Pull request by hzxie**
I created a script to update the `AUTHORS` file.
And I used this script to update the `AUTHORS` file.

The script (as you all know) comes from the @nodejs team.

### CODE REVIEW

File: scripts/update-authors.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  next if $seen{$_};
+  next if /\@chromium.org/;
+  next if /<erik.corry\@gmail.com>/;
+  $seen{$_} = push @authors, $_;
+}
+
+END {
+  print "# Authors ordered by first contribution.\n";
+  print "\n", @authors, "\n";
+  print "# Generated by tools/update-authors.sh\n";
```
Comment (beru) [inline scripts/update-authors.sh]: I think the file path in the comment should be updated to `scripts/update-authors.sh` to reflect current state.

Comment (hzxie) [reply scripts/update-authors.sh]: OMG.
I forgot.

File: AUTHORS [hunk 2]
```diff
@@ -73,3 +73,25 @@ MeiHui FAN <mhfan@ustc.edu>
...
+sid <srk97c@gmail.com>
+blackccpie <blackccpie@free.fr>
+Thomas van den Berg <thomas.g.vandenberg@gmail.com>
+Lakshay Garg <lakshayg@outlook.in>
+beru <berunpon@gmail.com>
+Junbo Zhang <dr.jimbozhang@gmail.com>
+edeforas <etienne.deforas@gmail.com>
+Haozhe Xie <cshzxie@gmail.com>
+
+# Generated by tools/update-authors.sh
```
[the remark this answers is not in the document]
Comment (hzxie) [reply AUTHORS]: The file is updated.
I'm sorry about making so many mistakes.

[the remark this answers is not in the document]
Comment (beru) [reply AUTHORS]: Thank you very much.
Please don't be sorry.

**Pull request merged**